### PR TITLE
Redefine roles

### PR DESCRIPTION
**Pull request by LoganHollins**
Closes bcjobs/analytics#209
Closes bcjobs/analytics#207
Closes bcjobs/analytics#208

### CODE REVIEW

File: app/components/employer-table/component.js [hunk 1]
```diff
@@ -5,8 +5,8 @@ var DATE_FORMAT = 'YYYY-MM-DD';
 
 export default Ember.Component.extend(ShowMore, {
   items: Ember.computed.alias('model.top'),
-  displayDownload: Ember.computed('exportSet', 'session.isLoggedIn', function(){
-    return this.get('session.isLoggedIn') && this.get('exportSet') !== undefined; // prevents download link from appearing if there is no endpoint
+  displayDownload: Ember.computed('exportSet', 'session.isAdmin', function(){
+    return this.get('session.isAdmin') && this.get('exportSet') !== undefined; // prevents download link from appearing if there is no endpoint
```
Comment (johnnyoshika) [inline app/components/employer-table/component.js:9]: @ryanstgermaine Do you still want any logged in user to see the `Download all records` option?  Or do you want this restricted to admins only?

image

**Comment by LoganHollins** [comment]
@johnnyoshika My bad, I asked Ryan and he said admins only.  I should of mentioned it in the PR.

File: app/components/employer-table/component.js (hunk 1 above)
Comment (ryanstgermaine) [inline app/components/employer-table/component.js:9]: @johnnyoshika I'd like it restricted to admins only now. It's a lot of data to give away and the amount of times BCIC logs in, I think we're ok. I can always make Bonnie an admin if need be or just send her a downloaded copy. They usually email me when they want those things anyway :).

Comment (johnnyoshika) [inline app/components/employer-table/component.js:9]: Got it, thanks!

**Pull request merged**